lib/gutdata/models/user_group.rb

```diff
@@ -54,18 +54,29 @@ def [](id, opts = {})
 
       # Create new user group
       #
+      # Should not be called directly. Use GutData::Project.add_user_group.
```

Comment:
Something that we may consider doing as well is backporting some of the new behavior in `models/project.rb` as well. Things like the added `Project#add_user_groups_to_dashboard` method may be of use to us as well

https://github.com/gooddata/gooddata-ruby/blob/42800640221cdb5bbb211aa8d286ac17fe14334c/lib/gooddata/models/project.rb#L479-L489

Reply:
Good call! Added in both the add/remove user groups to dashboard as well as the list user groups for dashboard methods to the Project model.